docs/content/preview/drivers-orms/csharp/postgres-npgsql.md

```diff
@@ -51,53 +50,57 @@ dotnet add package Npgsql
+NpgsqlConnection conn = new NpgsqlConnection(connStringBuilder)
 ```
 
 | Parameter | Description | Default |
 | :---------- | :---------- | :------ |
 | host  | Hostname of the YugabyteDB instance | localhost
 | port |  Listen port for YSQL | 5433
 | database | Database name | yugabyte
-| user | User for connecting to the database | yugabyte
+| user id| User for connecting to the database | yugabyte
 | password | Password for connecting to the database | yugabyte
 
-#### Using SSL
+#### Use SSL (Optional)
+
+Set up the driver properties to configure the credentials and SSL certificates for connecting to your cluster. The following table describes the additional parameters the .NET Npgsql driver requires as part of the connection string when using SSL.
 
-The .NET Npgsql driver validates certificates differently from other PostgreSQL drivers. When you specify SSL mode `require`, the driver verifies the certificate by default (like the `verify-ca` or `verify-full` modes), and fails for self-signed certificates (like YugabyteDB's). You can override this by specifying "Trust Server Certificate=true", in which case it bypasses walking the certificate chain to validate trust and hence works like other drivers' `require` mode. In this case, the Root-CA certificate is not required to be configured.
+| Npgsql Parameter | Description |
+| :---------- | :---------- |
+| Sslmode  | SSL Mode |
+| RootCertificate | Path to the root certificate on your computer |
 
-The following is an example connection string builder for connecting to a YugabyteDB cluster with SSL enabled.
+The following is an example connecting string parameter for connecting to YugabyteDB using SSL.
 
 ```csharp
 var connStringBuilder = new NpgsqlConnectionStringBuilder();
-connStringBuilder.Host = "22420e3a-768b-43da-8dcb-xxxxxx.aws.ybdb.io";
-connStringBuilder.Port = 5433;
-connStringBuilder.SslMode = SslMode.Require;
-connStringBuilder.Username = "admin";
-connStringBuilder.Password = "xxxxxx";
-connStringBuilder.Database = "yugabyte";
-connStringBuilder.TrustServerCertificate = true;
-CRUD(connStringBuilder.ConnectionString);
+    connStringBuilder.Host = "22420e3a-768b-43da-8dcb-xxxxxx.aws.ybdb.io";
+    connStringBuilder.Port = 5433;
+    connStringBuilder.SslMode = SslMode.VerifyFull;
+    connStringBuilder.RootCertificate = "/root.crt" //Provide full path to your root CA.
+    connStringBuilder.Username = "admin";
+    connStringBuilder.Password = "xxxxxx";
+    connStringBuilder.Database = "yugabyte";
+    CRUD(connStringBuilder.ConnectionString);
 ```
 
-| Parameter | Description | Default |
-| :---------- | :---------- | :------ |
-| sslmode  | SSL Mode | require
-| TrustServerCertificate |  Trust the server certificate configured on the YugabyteDB cluster | true
+If you created a cluster on [YugabyteDB Managed](https://www.yugabyte.com/managed/), use the cluster credentials and [download the SSL Root certificate](../../../yugabyte-cloud/cloud-connect/connect-applications/).
```

Comment:
[YugabyteDB Managed](https://www.yugabyte.com/managed/) clusters require SSL. Refer to [Connect applications](link) for instructions on how to obtain the cluster connection parameters and download the CA certificate.  
link to https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-connect/connect-applications/